app/main.py

```diff
@@ -1,6 +1,37 @@
+import os
+
+from dotenv import load_dotenv
+
+import requests
+
+load_dotenv()
+
+
+FILTERING = "Kyiv"
```

Comment:
The `FILTERING` constant is set to 'Kyiv', but according to the task description, it should be set to 'Paris' to fetch the weather data for Paris.